folium/map.py

```diff
@@ -228,33 +239,32 @@ class Marker(MacroElement):
     >>> Marker(location=[45.5, -122.3], popup=folium.Popup('Portland, OR'))
     # If the popup label has characters that need to be escaped in HTML
     >>> Marker(location=[45.5, -122.3],
-               popoup=folium.Popup('Mom & Pop Arrow Shop >>', parse_html=True))
+               popup=folium.Popup('Mom & Pop Arrow Shop >>', parse_html=True))
     """
     _template = Template(u"""
-            {% macro script(this, kwargs) %}
-
-            var {{this.get_name()}} = L.marker(
-                [{{this.location[0]}}, {{this.location[1]}}],
-                {
-                    icon: new L.Icon.Default()
-                    }
-                )
-                {% if this.tooltip %}.bindTooltip("{{this.tooltip.__str__()}}"){% endif %}
-                .addTo({{this._parent.get_name()}});
-            {% endmacro %}
-            """)
+        {% macro script(this, kwargs) %}
+        var {{this.get_name()}} = L.marker(
+            [{{this.location[0]}}, {{this.location[1]}}],
+            {
+                icon: new L.Icon.Default()
+                }
+            ).addTo({{this._parent.get_name()}});
+        {% endmacro %}
+        """)
 
-    def __init__(self, location, popup=None, tooltip=None, icon=None):
+    def __init__(self, location=None, popup=None, tooltip=None, icon=None):
```

Comment:
Changing the default of `location` here doesn't have to do with tooltip, please remove it here.